### PR TITLE
Add option - filter is not scalable

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/rebloom.c [hunk 1]
```diff
@@ -83,8 +84,9 @@ static const char *statusStrerror(int status) {
  * Common function for adding one or more items to a bloom filter.
  * capacity and error rate must not be 0.
  */
-static SBChain *bfCreateChain(RedisModuleKey *key, double error_rate, size_t capacity, unsigned expansion) {
-    SBChain *sb = SB_NewChain(capacity, error_rate, BLOOM_OPT_FORCE64, expansion);
+static SBChain *bfCreateChain(RedisModuleKey *key, double error_rate,
+                              size_t capacity, unsigned expansion, unsigned scaling) {
```
Comment (gkorland) [inline src/rebloom.c]: here scaling is unsigned

Comment (ashtul) [reply src/rebloom.c]: @gkorland 
Redis does only RedisModule_StringToLongLong but _options_ in _SBChain_ is unsigned.

Comment (ashtul) [reply src/rebloom.c]: @gkorland 
OK. That was silly.
Fixed

**Comment by ashtul** [comment]
#158

**Pull request merged**